crates/relayer/src/chain/cosmos.rs

```diff
@@ -1912,9 +1981,31 @@ fn do_health_check(chain: &CosmosSdkChain) -> Result<(), Error> {
         );
     }
 
+    let relayer_gas_price = &chain.config.gas_price;
+    let node_min_gas_prices = chain.min_gas_price()?;
+    let mut found_matching_denom = false;
+
+    for price in node_min_gas_prices {
+        match price.partial_cmp(relayer_gas_price) {
+            Some(Ordering::Greater) => return Err(Error::gas_price_too_low(chain_id.clone())),
```

Comment:
I would invert the condition here to make it slightly more consistent with the euro that's returned, eg.

```suggestion
        match relay_gas_price.partial_cmp(price) {
            Some(Ordering::Lesser) => return Err(Error::gas_price_too_low(chain_id.clone())),
```